man/systemctl.xml

```diff
@@ -1502,6 +1502,18 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-P</option></term>
+        <term><option>--property-value=</option></term>
+
+        <listitem>
+          <para>This is similar to <option>-p</option>/<option>--property=</option> above, but skips the
+          property name <literal>=</literal>, as if <option>--value</option> was used. Note that using
+          <option>-P</option>/<option>--property-value=</option> once will also affect all proprties
+          listed with <option>-p</option>/<option>--property=</option>.</para>
```

Comment:
Why not just say "-P is equivalent to --value --property="? Sounds so much shorter.